kokoro/config/build/presubmit/plucky_aarch64.gcl

```diff
@@ -0,0 +1,10 @@
+import '../common.gcl' as common
```

Comment:
Please remove all changes to the `kokoro/` folder in this PR. They will be done in a followup update.